core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpActivity.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.help
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import androidx.recyclerview.widget.DividerItemDecoration
+import org.kiwix.kiwixmobile.core.R
+import org.kiwix.kiwixmobile.core.base.BaseActivity
+import org.kiwix.kiwixmobile.core.di.components.CoreComponent
+import org.kiwix.kiwixmobile.core.utils.Constants
+import org.kiwix.kiwixmobile.core.utils.LanguageUtils.Companion.getCurrentLocale
+import kotlinx.android.synthetic.main.activity_help.activity_help_feedback_image_view
+import kotlinx.android.synthetic.main.activity_help.activity_help_feedback_text_view
+import kotlinx.android.synthetic.main.activity_help.activity_help_recycler_view
+import kotlinx.android.synthetic.main.layout_toolbar.toolbar
+
+class HelpActivity : BaseActivity() {
+  private val titleDescriptionMap by lazy {
+    hashMapOf(R.string.help_2 to R.array.description_help_2,
+              R.string.help_5 to R.array.description_help_5
+    ).entries.associateTo(hashMapOf<String, String>(),
```

Comment:
we can improve the formatting of this a bit, with a sequence of method calls they should go on a new line.

I am a bit surprised ktlint allowed this but the build did pass.

Also I might have been misguided telling you to start with a map, if you start with a `listOf` our existing pairs then you can use `associate` to create our desired map and that will cut down on the operators here.

Also prefer mapOf instead of `hashMapOf`, we generally don't want to leak the underlying implementation of map

Reply:
Ok, I will try that.